src/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -411,14 +411,12 @@ private Interop.User32.WNDCLASS WndClass
             {
                 if (s_staticwndclass == null)
                 {
-                    const string classNameFormat = ".NET-BroadcastEventWindow.{0}.{1}";
-
                     IntPtr hInstance = Interop.Kernel32.GetModuleHandle(null);
 
-                    s_className = string.Format(System.Globalization.CultureInfo.InvariantCulture,
-                        classNameFormat,
-                        Convert.ToString(AppDomain.CurrentDomain.GetHashCode(), 16),
-                        s_domainQualifier);
+                    s_className = string.Format(
```

Comment:
Can't use `SR.Format` here? Should we add a TODO or comment with an issue?

Reply:
SR.Format is for formatting SR resources... this isn't one.